### PR TITLE
[SYCL][Joint Matrix][E2E] Update tests to require certain Intel GPU driver version

### PR DESCRIPTION
**Pull request by YuriPlyakhin**

### CODE REVIEW

File: sycl/test-e2e/Matrix/SG32/element_wise_abc.cpp [hunk 1]
```diff
@@ -6,6 +6,7 @@
 //
 //===----------------------------------------------------------------------===//
 // REQUIRES: matrix
+// REQUIRES-INTEL-DRIVER: lin: 27501, win: 101.4943
```
Comment (dkhaldi) [inline sycl/test-e2e/Matrix/SG32/element_wise_abc.cpp:9]: Is the expectation that these will be ignored on the CPU?

**Review by dkhaldi — COMMENTED** [review]
Can you also add this to Matrix~joint_matrix_transposeC.cpp?
I see you only added it to the SG32/ version

**Comment by YuriPlyakhin** [comment]
> Can you also add this to Matrix~joint_matrix_transposeC.cpp? I see you only added it to the SG32/ version

Sorry, I can not. This IGC feature did not reach Agama/NEO builds yet, so I don't know the version to put into the check.

**Review by dkhaldi — APPROVED** [review]
LGTM

**Comment by YuriPlyakhin** [comment]
@intel/llvm-gatekeepers , please, merge. all green.

**Pull request merged**